packages/components/src/DataViewer/theme.scss

```diff
@@ -0,0 +1,56 @@
+$tc-layout-skeleton-width: 9.6rem !default;
+$tc-layout-skeleton-height: 9.6rem !default;
+
+.tc-twoviewers-layout {
+	display: flex;
+	width: 100%;
+	overflow: hidden;
+
+	&-loading {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+
+		:global(.tc-svg-icon) {
+			width: $tc-layout-skeleton-width;
+			height: $tc-layout-skeleton-height;
+		}
+	}
+
+	&-left,
+	&-right {
+		background-color: $wild-sand;
+	}
+
+	&-left {
+		border-right: 0.5rem solid white;
+		width: 47rem;
+	}
+
+	&-right {
+		flex-grow: 1;
+	}
+}
+
+.tdc {
+	:global {
+		@import '~@talend/bootstrap-theme/src/theme/variations/tdc';
+	}
+}
+
+:global(.sb-show-main) {
+	background: #f6f6f6;
```

Comment:
Wat?

Reply:
Same as above, only used in the story. Was copied from the original story, seems to enhance the display in storybook. 🤷‍♂️

Reply:
no! by using `:global` here, you will change ALL stories background

Reply:
It was to highlight the semantic chooser i think on ui-dataset storybook.
We may remove this specific part i think

Reply:
Removed